docs/1.concepts/abstraction/chain-signatures.md

```diff
@@ -4,22 +4,23 @@ title: Chain Signatures
 sidebar_label: What are Chain Signatures?
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+Chain Signatures unlock the ability for a single account to transact across multiple blockchain protocols, giving ownership of cross-chain accounts, data, and assets to one NEAR account.
 
-Chain Signatures unlock the ability for a single account to transact across multiple blockchain protocols, giving ownership of many accounts, data, and assets to one NEAR account. 🤯
+This many-to-one ownership is made possible through a mixture of services across our tech stack:
 
-This many-to-one ownership is made possible through NEAR's [unique account model](../basics/accounts/introduction.md), which allows many keys to be associated with a single account. Chain Signatures generates and uses one of these keys to sign payloads compatible with other blockchain platforms but still linked to a native NEAR account.
+1. A [smart contract](../basics/accounts/smartcontract.md) that holds requests for multi-chain signatures.
+2. A [multiparty computation](https://www.zellic.io/blog/mpc-from-scratch/) service handling user's keys and listening for requests to sign payloads.
+3. A multi-chain [relayer](./relayers.md), which can submit signed transactions to other networks.
 
 ---
 
 ## How It Works
 
-Chain Signatures in four basic steps:
+There are four steps involved on Chain Signatures:
 
-1. [Create Payload](#1-create-a-payload) - A payload is created to be signed and sent to a given blockchain platform.
-2. [Signature Request](#2-request-signature) - A smart contract call is made to sign the payload.
-3. [MPC Signing Service](#3-sign-with-mpc) - A service signs the payload linking it with the sender's NEAR account.
+1. [Create a Transaction](#1-create-a-payload) - The user creates the transaction they intent to submit to another blockchain
```

Comment:
@gagdiez, I used payload here as users are not limited to this one action. Messages, data, identity verification, etc. are also valid use cases. I understand that you are most likely going for simplicity here but worry we might be not technically correct or introducing a perceived limitation to devs. 

@volovyks @DavidM-D what are your thoughts on using transactions over payload?

Reply:
while "payload" is technically correct, I think that transaction is easier to be understood. Thought, we can switch it back if we are going for correctness

Reply:
Yeah, certainly a fine line to dance here between "technically" vs "easier to comprehend". 
@mattlockyer @kcole16 wdyt is better for EthDevs? Is payload confusing? Should we just use transactions for simplicity / better understanding?